modules/gui/src/com/haulmont/cuba/gui/components/filter/SaveFilterWindow.java

```diff
@@ -56,7 +56,7 @@ public void init(Map<String, Object> params) {
         if (!Strings.isNullOrEmpty(filterNameParam)) {
             filterName.setValue(filterNameParam);
         }
-        MetaProperty property = metadata.getClassNN(FilterEntity.class).getProperty("name");
+        MetaProperty property = metadata.getClassNN(FilterEntity.class).getPropertyNN("name");
         if (property != null) {
```

Comment:
`if property != null` isn't required if you use `getPropertyNN`. See javadoc: `Throws exception if not found.`